src/nvim/tui/tui.c

```diff
@@ -1563,6 +1560,10 @@ static void patch_terminfo_bugs(TUIData *data, const char *term,
   bool mate_pretending_xterm = xterm && colorterm
     && strstr(colorterm, "mate-terminal");
   bool true_xterm = xterm && !!xterm_version && !bsdvt;
+  bool true_screen = screen && !os_getenv("TMUX");
```

Comment:
do we really need `true_screen` ? I mean if TMUX is set while running screen, aren't you screwed anyway ?

Reply:
For `tmux` with `screen*` set in `$TERM`, `Ss/Se` etc. must output the `tmux` sequence(`CSI[%p1%d q/CSI[ q`) and only `sitm` must be treated like screen. So the name does not have to be `true_screen`, but it must identify the `tmux` and true `GNU Screen`.